### PR TITLE
PUD-1268  Add missing document record tests to PACT

### PR DESCRIPTION
**Pull request by AndyNewton**
PUD-1268  Add missing document record tests to PACT: API side, needs to precede UI side change.

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt [hunk 1]
```diff
@@ -193,45 +197,59 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
     )
   }
 
+  // TODO: delete as not in use?
```
Comment (philsegal) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt:200]: delete them :)

**Review by philsegal — DISMISSED** [review]
I'd bin off the unused states.

File: helm_deploy/manage-recalls-api/values.yaml [hunk 2]
```diff
@@ -25,7 +25,10 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     GOTENBERG_ENDPOINT_URL: http://manage-recalls-api-gotenberg
-    SENTRY_IGNORED_EXCEPTIONS_FOR_TYPE: org.springframework.security.web.firewall.RequestRejectedException,uk.gov.justice.digital.hmpps.managerecallsapi.service.VirusFoundException
+    SENTRY_IGNORED_EXCEPTIONS_FOR_TYPE:
```
Comment (philsegal) [inline helm_deploy/manage-recalls-api/values.yaml:28]: That format works?

**Review by philsegal — APPROVED** [review]

File: helm_deploy/manage-recalls-api/values.yaml (hunk 2 above)
Comment (AndyNewton) [reply helm_deploy/manage-recalls-api/values.yaml:28]: @dazoakley As YAML guru does ^^^ change I've made look valid to you please?  My IDE is telling me it is correct as is various googling, i.e. this is a valid YAML Sequence which evaluates to the same object as an Array?  Ta.

Comment (AndyNewton) [reply helm_deploy/manage-recalls-api/values.yaml]: (Force pushed) latest commit to switch this to a single line per chat with @dazoakley : https://mojdt.slack.com/archives/C02ET0U1G4T/p1641899860007400?thread_ts=1641871219.002700&cid=C02ET0U1G4T

**Review by dazoakley — APPROVED** [review]

**Pull request merged**